tests/test_cases.yaml

```diff
@@ -136,6 +136,32 @@ tests:
       stderr:
         - 'unknown command "non-existent" for "atmos"'
       exit_code: 1
+  - name: atmos validate editorconfig
+    enabled: true
+    description: "Ensure atmos CLI validates success case"
+    workdir: "../examples/quick-start-simple/"
+    command: "atmos"
+    args:
+      - "validate"
+      - "editorconfig"
+    expect:
+      stdout:
+        - "No errors found"
+      stderr:
+        - "^$"
+      exit_code: 0
+  - name: atmos validate editorconfig error
+    enabled: true
+    description: "Ensure atmos CLI gives error during validation based on editorconfig"
+    workdir: "../examples/quick-start-advanced/"
```

Comment:
Heh, it hurts that we are using a quick star for a negative case, when instead we should fix the quick start. Instead let's use example/tests, which is expected to have errors for testing.